File: 5032.c
#include <stdio.h>
int main(){
    int e,f,c;
    scanf("%d %d %d", &e, &f, &c);
    e=e+f;
    int count=0;
    while(1){
        if (e<c) break;
        e-=c;
        e+=1;
        count+=1;
    }
    printf("%d", count);
    return 0;
}

File: 5032.py
#1. 바꾼다
#2. 마신다
#3. 쌓인다
e,f,c=map(int,input().split())
e=e+f
count=0
while True:
    if e<c: break
    e-=c
    e+=1
    count+=1
print(count)